src/pages/[platform]/how-amplify-works/faq/index.mdx

```diff
@@ -29,42 +29,43 @@ export function getStaticProps(context) {
+
+**In Gen 1, Amplify offered a set of use case categories for building applications (for example, Authentication, Analytics, API, DataStore, Geo, and Predictions). Are those same categories available in Gen 2?**
 
-While Gen 2 is in public preview, we will provide migration guides that describe how to manually move from Gen 1 to Gen 2. When generally available, we will offer tooling that will help developers move projects from Gen 1 to Gen 2.
+Amplify code-first DX (Gen 2) supports Auth, Data, Storage, and Functions. Other use cases can be implemented using AWS Cloud Development Kit (AWS CDK) constructs which there is documentation for under the respective category name.
 
-**What platforms will Gen 2 support?** 
+**What if we want to add a feature like AI/ML or Amazon Location Service to our application in Gen 2?**
 
-While in public preview, Gen 2 supports a TypeScript-first developer experience for building backends. Teams that have built backends with Gen 2 can reuse that backend in mobile apps if desired.
+Because Amplify builds on the AWS CDK, any of the 175+ AWS services can be added to your app using [custom resources](/[platform]/build-a-backend/add-aws-services/custom-resources/) and L2/L1 AWS CDK constructs.
 
-**Which web frameworks will have first-class support?** 
+**What happens once my application grows too big and I want to do more configuration with my application (add more features, other AWS services, etc.)?*
 
-Gen 2 offers support for all web frameworks that are supported in the Amplify libraries (currently Next.js, React, Vue, and Angular).
+Amplify is built on top of the AWS CDK, so you can add any [AWS service to your Amplify app](/[platform]/build-a-backend/add-aws-services/custom-resources/). You can also override [Amplify-generated configuration of your resources](/[platform]/build-a-backend/add-aws-services/custom-resources/) using the CDK.
 
-**Does Gen 2 support mobile development?**
+**How much does it cost to operate Amplify Gen2?**
 
-You can use a Gen 2 backend with a Swift, Kotlin, or Flutter app; however, in public preview, Amplify only supports TypeScript.
+You can read all about Amplify's pricing on our [pricing page](https://aws.amazon.com/amplify/pricing/).
 
-**What if I'm already building with AWS CDK? How should I proceed?**
+**Can I use any other database storage other than Amazon DynamoDB? Does it have to be noSQL?**
 
-If you already have an AWS CDK project, you can use the L3 constructs that Amplify provides. If you're starting a new project with Auth and Data, then start with Amplify because you can always use AWS CDK with it.
+You can also use an [existing MySQL or PostgreSQL database](/react/build-a-backend/graphqlapi/connect-api-to-existing-database/) in an Amplify app.
```

Comment:
Sorry @aspittel, just realized, should this link be platform agnostic or should it link to the `react` page?

Reply:
Updated to [platform]!